how-to-guides/neptune-docker/scripts/training.py

```diff
@@ -0,0 +1,93 @@
+import torch
+import torch.nn as nn
+import torch.optim as optim
+from torchvision import datasets, transforms
+import neptune.new as neptune
+import os
+
+
+# read docker secret as environment
+with open(os.environ['NEPTUNE_API_TOKEN']) as file:
+    neptune_token = file.read() 
+
+# Step 1: Initialize Neptune and create new Neptune Run
+run = neptune.init(
+    project="common/pytorch-integration",
+    api_token=neptune_token,
+    tags="Neptune Docker"
+)
+
+data_dir = "data/CIFAR10"
+compressed_ds = "./data/CIFAR10/cifar-10-python.tar.gz"
+data_tfms = {
+    "train": transforms.Compose(
+        [
+            transforms.RandomHorizontalFlip(),
+            transforms.ToTensor(),
+            transforms.Normalize([0.485, 0.456, 0.406], [0.229, 0.224, 0.225]),
+        ]
+    )
+}
+
+params = {
+    "lr": 1e-2,
+    "batch_size": 128,
+    "input_size": 32 * 32 * 3,
+    "n_classes": 10,
+    "model_filename": "basemodel",
+}
+
+
+class BaseModel(nn.Module):
+    def __init__(self, input_sz, hidden_dim, n_classes):
```

Comment:
If you approved my suggestion, pls make sure that script will work -> here you have `input_sz`, not `input_size`

Reply:
It works, 

But I'm making it seamless with one naming convetion